api/release-notes/release-notes-202311.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 2
+---
+
+# iCure API version 202311.0.0
+
+This iCure release is a long term support release. It is going to be available at https://v202311.api.icure.cloud for at least 30 months, from the 15th of November 2023 to the 15th of March 2026.
+This version has the commit code 4.0.489-g3c3d708f38. 
+
+### Breaking Changes
+
+- This release introduces stricter control accesses on endpoints that have a security impact or can affect destructively the data. The iCure API will respond with a 401 if the level of authentication is not high enough.
+- This release supports a new format for the storage of encrypted keys in shareable entities (Patient, Health Elements, Contacts, …) that allows for anonymous sharing of data with patients. When a piece of data is created with this new format it will not be available from previous versions of the API.
+
+### New Features
+
+- Agenda : New anonymous endpoints for the collection of public agenda information
+- Agenda : Support of RRule format for the definition of the time slots
+- Messaging : Support for Sendgrid, Google, FriendlyCaptcha & Twilio accounts setup in messaging gateway
+- Searching and querying : new filters for Contacts, Healthcare Elements, Patients and Messages
+- Terminologies and codifications: Support for Snomed CT
+- Terminologies and codifications: Support for LOINC
+- Terminologies and codifications: Support for the querying of codes per version
+- Cryptographic model : Support for secure delegations that allows for anonymized sharing of information to the patient
+- Cryptographic model : Better control of trusted keys
+- Cryptographic model : Automated recovery of lost keys
+- Authentication and Authorisation: JWT tokens support
+- Authentication and Authorisation: Authentication with OAuth
+- Authentication and Authorisation: Authentication by email or SMS tokens
+- Authentication and Authorisation: Control of accesses through role system
+- Authentication and Authorisation: Support for database quotas
```

Comment:
I would remove references to features that do not apply to the lite apis, such as agenda.

Reply:
I think that at some point we might want to include the agenda in the lite SDKs, possibly before the end of support of this release